### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - David Schildböck

### DISCUSSION THREAD

**Discussion by nyxhawk** [heig-vd-dai-course/heig-vd-dai-course#344] [Show and tell]
Mon repo [Github](https://github.com/shadowkudo/DAI-04)

**Comment by ludelafo** [comment]
Looks good to me, thanks!

One tip: you can use code blocks to format code as mentioned in the course material: https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#code-blocks or `inline code` with the command you want to highlight between `` ` `` (backtick) characters.

It makes the reading of your commands easier. :)
